src/mono/mono/utils/w32subset.h

```diff
@@ -358,67 +187,6 @@
 #endif
 #endif
 
-#ifndef HAVE_API_SUPPORT_WIN32_LOCAL_INFO
```

Comment:
@lateralusX I assume all this kind of stuff is in System.Native on the none-Windows port?

Reply:
When we moved over to dotnet/runtime most of the runtime code guarded with these defines where removed and replaced with calls in BCL, so if code in runtime is not guarded with these defines there is nothing downstream that could take advantage of it anymore and if it still needs ways to include/exclude source, it will define its own include/exclude logic.